google/cloud/documentai_toolbox/wrappers/document.py

```diff
@@ -582,7 +559,7 @@ def get_form_field_by_name(self, target_field: str) -> List[FormField]:
         return found_fields
 
     def form_fields_to_dict(self) -> Dict:
-        r"""Returns Dictionary of form fields in document.
+        r"""Returns dictionary of form fields in document.
```

Comment:
Make the type annotation (and maybe the docstring too) more informative.  Is the return type `Dict[str, str]` here?

Reply:
Added a more explicit type annotation `Dict[str, Union[str, List[str]]]`